protocol/consistencychecks.go

```diff
@@ -220,7 +220,11 @@ func (cc *ConsistencyChecks) verifyAuthPath(uname string, key []byte,
 	return proofType, nil
 }
 
-func (cc *ConsistencyChecks) verifySTR(str *m.SignedTreeRoot) error {
+// verifySTR checks the consistency between 2 snapshots.
+// This uses the pinning signing key in cc
+// to verify the STR's signature and should not verify
+// the hash chain using the current STR stored in cc.
```

Comment:
Just to clarify, `verifySTR` shouldn't verify the hash chain using `cc.SavedSTR` in order to support multi-epoch checks (as in `KeyLookupInEpoch`)?